plugin:health SIGUSR1: listen tcp :8080: bind: address already in use

Observed for CoreDNS 1.1.2 with Corefile
```
.:53 {
	health
	forward . 8.8.8.8:53
	errors
}
```
some graceful restarts failed:
```
.:53
2018/04/23 17:52:57 [INFO] CoreDNS-1.1.2
2018/04/23 17:52:57 [INFO] linux/amd64, go1.9.2, 
CoreDNS-1.1.2
linux/amd64, go1.9.2, 
2018/04/23 17:53:14 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:14 [INFO] Reloading
2018/04/23 17:53:14 [ERROR] SIGUSR1: listen tcp :8080: bind: address already in use
2018/04/23 17:53:16 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:16 [INFO] Reloading
2018/04/23 17:53:16 [INFO] Reloading complete
2018/04/23 17:53:18 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:18 [INFO] Reloading
2018/04/23 17:53:18 [INFO] Reloading complete
2018/04/23 17:53:19 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:19 [INFO] Reloading
2018/04/23 17:53:19 [INFO] Reloading complete
2018/04/23 17:53:20 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:20 [INFO] Reloading
2018/04/23 17:53:20 [ERROR] SIGUSR1: listen tcp :8080: bind: address already in use
2018/04/23 17:53:21 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:21 [INFO] Reloading
2018/04/23 17:53:21 [INFO] Reloading complete
2018/04/23 17:53:23 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:23 [INFO] Reloading
2018/04/23 17:53:23 [INFO] Reloading complete
2018/04/23 17:53:24 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:24 [INFO] Reloading
2018/04/23 17:53:24 [INFO] Reloading complete
2018/04/23 17:53:25 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:25 [INFO] Reloading
2018/04/23 17:53:25 [ERROR] SIGUSR1: listen tcp :8080: bind: address already in use
2018/04/23 17:53:26 [INFO] SIGUSR1: Reloading
2018/04/23 17:53:26 [INFO] Reloading
2018/04/23 17:53:26 [INFO] Reloading complete
```

--- a/plugin/health/README.md
+++ b/plugin/health/README.md
@@ -48,6 +48,16 @@ net {
 }
 ~~~
 
+Note this is also true if you format your Corefile as:
+
+~~~ corefile
+com net {
+    whoami
+    erratic
+    health :8080
+}
+~~~~
+
 ## Plugins
 
 Any plugin that implements the Healther interface will be used to report health.

--- a/plugin/health/README.md
+++ b/plugin/health/README.md
@@ -48,9 +48,10 @@ net {
 }
 ~~~
 
-Note this is also true if you format your Corefile as:
+Note that if you format this in one server block you will get an error on startup, that the second
+server can't setup the health plugin (on the same port).
 
-~~~ corefile
+~~~ txt
 com net {
     whoami
     erratic

